Day to day work/Project documentation.md

```diff
@@ -39,5 +39,5 @@ Some of the examples:
 Anyone working on the project (i.e., Developers, Designers, QA, PM, etc.) should participate in creating this document. 
 This is the only documentation that is not defined by technology. So feel free to write it on a separate Wiki (GitHub) repo, Confluence, Productive, Google Docs, etc.
 
-
-
+---
+*In case you need some examples on how each level of documentation looks like in real life projects, please contact a proejct team that has such level of documetnation that interests you. For example, a good reference point to see this for a particular mobile projects is to check the Mobile Project Health Score sheet.*
```

Comment:
Will the project health score be something public or internal? If we do not publish this publicly, consider removing the second sentence.

Reply:
Probably not, but I would nevertheless leave it here because it will give a reference for our internal people to know where to look at. For external people it will be a mystery, but that good kind of mystery; oh look they track health scores of their projects :D